portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-cell-validators.js

```diff
@@ -0,0 +1,200 @@
+      column: facilityIdObject?.column,
+      row: facilityIdObject?.row || rowNumber,
+      value: facilityIdObject?.value,
+      exporter: exporterName,
+    };
+  }
+  return null;
+};
+
+const generateBaseCurrencyError = (baseCurrencyObject, exporterName, rowNumber) => {
+  if (!baseCurrencyObject?.value) {
+    return {
+      errorMessage: 'Base currency must have an entry',
+      column: baseCurrencyObject?.column,
+      row: baseCurrencyObject?.row || rowNumber,
+      value: baseCurrencyObject?.value,
+      exporter: exporterName,
+    };
+  }
+  if (!validator.isISO4217(baseCurrencyObject?.value)) {
+    return {
+      errorMessage: 'Base currency must be in the ISO 4217 currency code format',
+      column: baseCurrencyObject?.column,
+      row: baseCurrencyObject?.row || rowNumber,
+      value: baseCurrencyObject?.value,
+      exporter: exporterName,
+    };
+  }
+  return null;
+};
+
+const generateFacilityUtilisationError = (facilityUtilisationObject, exporterName, rowNumber) => {
+  if (!facilityUtilisationObject?.value) {
+    return {
+      errorMessage: 'Facility utilisation must have an entry',
+      column: facilityUtilisationObject?.column,
+      row: facilityUtilisationObject?.row || rowNumber,
+      value: facilityUtilisationObject?.value,
+      exporter: exporterName,
+    };
+  }
+  if (!CURRENCY_NUMBER_REGEX.test(facilityUtilisationObject?.value)) {
+    return {
+      errorMessage: 'Facility utilisation must be a number',
+      column: facilityUtilisationObject?.column,
+      row: facilityUtilisationObject?.row || rowNumber,
+      value: facilityUtilisationObject?.value,
+      exporter: exporterName,
+    };
+  }
+  if (facilityUtilisationObject?.value?.length > 15) {
+    return {
+      errorMessage: 'Facility utilisation must be 15 characters or less',
+      column: facilityUtilisationObject?.column,
+      row: facilityUtilisationObject?.row || rowNumber,
+      value: facilityUtilisationObject?.value,
+      exporter: exporterName,
+    };
+  }
+  return null;
+};
+
+const generateTotalFeesAccruedError = (totalFeesAccruedObject, exporterName, rowNumber) => {
+  if (!totalFeesAccruedObject?.value) {
+    return {
+      errorMessage: 'Total fees accrued for the month must have an entry',
+      column: totalFeesAccruedObject?.column,
+      row: totalFeesAccruedObject?.row || rowNumber,
+      value: totalFeesAccruedObject?.value,
+      exporter: exporterName,
+    };
+  }
+  if (!CURRENCY_NUMBER_REGEX.test(totalFeesAccruedObject?.value)) {
+    return {
+      errorMessage: 'Total fees accrued for the month must be a number',
+      column: totalFeesAccruedObject?.column,
+      row: totalFeesAccruedObject?.row || rowNumber,
+      value: totalFeesAccruedObject?.value,
+      exporter: exporterName,
+    };
+  }
+  if (totalFeesAccruedObject?.value?.length > 15) {
+    return {
+      errorMessage: 'Total fees accrued for the month must be 15 characters or less',
```

Comment:
It might be worth making the 15 a more prominent constant? You can re-use it in the error message as well